internal/pkg/argocd/argocd.go

```diff
@@ -53,7 +53,7 @@ type DiffResult struct {
 
 // Mostly copied from  https://github.com/argoproj/argo-cd/blob/4f6a8dce80f0accef7ed3b5510e178a6b398b331/cmd/argocd/commands/app.go#L1255C6-L1338
 // But instead of printing the diff to stdout, we return it as a string in a struct so we can format it in a nice PR comment.
```

Comment:
Maybe we want to update this comment too as the function signature is also different now.